posthog/migrations/0155_insightcomment.py

```diff
@@ -0,0 +1,40 @@
+# Generated by Django 3.1.8 on 2021-05-19 00:44
+
+import django.db.models.deletion
+from django.conf import settings
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0154_dashboarditem_parent"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
```

Comment:
You should be able to copy this function into the previous migration to avoid adding multiple migrations in one go